server/backend/database/database.go

```diff
@@ -138,7 +138,9 @@ type Database interface {
 	FindDeactivateCandidates(
 		ctx context.Context,
 		candidatesLimitPerProject int,
-	) ([]*ClientInfo, error)
+		projectFetchSize int,
+		housekeepingLastProjectID types.ID,
```

Comment:
In a layered architecture, it is recommended that the callee doesn't know about the caller.

`FindDeactivateCandidates` is only called by `housekeeping` package for now but `database` can be referenced by other packages too. And I think `housekeepingLastProjectID` has a semantic dependency on the caller, `housekeeping` package.

How about removing `housekeeping` prefix from the variable in `database` package?